resources/replace_or_add.rb

```diff
@@ -5,6 +5,7 @@
 property :path, String
 property :pattern, [String, Regexp]
 property :replace_only, [true, false]
+property :remove_duplicates, [true, false]
```

Comment:
I would explicitly set the default to false for clarity and update the documentation to match the new default